src/core/ext/filters/client_channel/lb_policy/pick_first/pick_first.cc

```diff
@@ -336,6 +329,12 @@ absl::Status PickFirst::UpdateLocked(UpdateArgs args) {
           DisableOutlierDetectionAttribute::kName,
           std::make_unique<DisableOutlierDetectionAttribute>()));
     }
+    if (args.config != nullptr) {
+      auto config = static_cast<PickFirstConfig*>(args.config.get());
```

Comment:
I'd rather not put this code in this block, because this block is going to go away in the not-too-distant future, as per the TODO comment on line 321.  Instead, please put this code in an `else` clause on line 320.  (Sorry, looks like I wrote the wrong line number in my previous comment.)